src/components/nodes/signalEndEvent/signalEndEvent.vue

```diff
@@ -1,21 +1,17 @@
 <script>
 import EndEvent from '@/components/nodes/endEvent/endEvent';
 import signalEndEventIcon from '@/assets/signal-end-event.svg';
-import pull from 'lodash/pull';
 
 export default {
   extends: EndEvent,
   props: ['rootElements'],
```

Comment:
Looks like you forgot this. The `rootElements` props isn't used anymore now and can be removed.
```suggestion
```